vllm/v1/serial_utils.py

```diff
@@ -133,9 +133,26 @@ def _encode_ndarray(
         # backing buffers that we've stashed in `aux_buffers`.
         return obj.dtype.str, obj.shape, data
 
+    def _encode_tensor(
+        self, obj: torch.Tensor
+    ) -> tuple[str, tuple[int, ...], Union[int, memoryview]]:
+        assert self.aux_buffers is not None
+        # this creates a copy of the tensor
+        obj = obj.contiguous() if not obj.is_contiguous() else obj
```

Comment:
This will return the same tensor if it's already contiguous (sorry missed first time around)
```suggestion
        obj = obj.contiguous()
```